policyengine_us/tests/policy/baseline/gov/states/ga/tax/income/subtractions/ga_retirement_exclusion.yaml

```diff
@@ -0,0 +1,61 @@
+- name: head is disabled
+  period: 2023
+  input:
+    people:
+      person1:
+        is_tax_unit_head: true
+        is_disabled: true
+        age: 50
+        ga_retirement_income: 69_000
+    tax_units:
+      tax_unit:
+        members: [person1]
+        filing_status: SINGLE
+    households:
+      household:
+        state_code: GA
+  output:
+    ga_retirement_exclusion: 35_000
+
+- name: head not eligible
+  period: 2023
+  input:
+    people:
+      person1:
+        is_tax_unit_head: true
+        is_disabled: false
+        age: 50
+        ga_retirement_income: 69_000
+    tax_units:
+      tax_unit:
+        members: [person1]
+        filing_status: SINGLE
+    households:
+      household:
+        state_code: GA
+  output:
+    ga_retirement_exclusion: 0
+
+- name: head is eligible for older cap and spouse is eligible for younger cap
```

Comment:
make another tests with multiple people qualifying for the exclusion (one below cap and one above) etc.